src/components/Voting.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { PatchArticleVotes } from "./api-logic";
+
+const Voting = ({ SingleArticle, article_id }) => {
+  const [vote, setVote] = useState(0);
+  const [appendedVote, setAppendedVote] = useState(SingleArticle.votes);
+  useEffect(() => {
+    PatchArticleVotes(article_id, vote)
```

Comment:
We should also handle an error coming back from this. Think about how you as a user would like to be notified if the request had failed.